src/status_im/contexts/settings/wallet/saved_addresses/add_address_to_save/view.cljs

```diff
@@ -144,9 +144,9 @@
         on-change-text                      (rn/use-callback
                                              (fn [new-value]
                                                (let [trimmed-value (string/trim new-value)]
-                                                 (set-error (validate (string/lower-case trimmed-value)))
                                                  (set-address-or-ens trimmed-value)
                                                  (set-ens-address "")
+                                                 (set-error (validate (string/lower-case trimmed-value)))
```

Comment:
A minor change in order of set-state. 

This fixes an non-fatal crash if you type/paste an existing address, the error state is updated which triggers the re-render (for error view) before the actual address text is set in the state. The error state should be validated after the values are set.